src/Enterspeed.Cli/Commands/Schema/CreateSchemaCommand.cs

```diff
@@ -42,6 +42,7 @@ public new class Handler : BaseCommandHandler, ICommandHandler
             public string Alias { get; set; }
             public string Name { get; set; }
             public string Type { get; set; }
+            public string Format { get; set; }
```

Comment:
There is not command option to set the schema format

Reply:
This has been added. Could you verify that it works?

Reply:
Yes, it works now.